examples/tensorflow/train_predict.py

```diff
@@ -2,53 +2,69 @@
 from __future__ import division
 from __future__ import print_function
 
+import argparse
 import mlflow
 from mlflow import tensorflow, tracking, pyfunc
 import numpy as np
 import pandas as pd
 import shutil
 import tempfile
 import tensorflow as tf
+import os.path
 
+# Command-line arguments
+parser = argparse.ArgumentParser(description='Tensorflow Example')
+# parser.add_argument('--hidden_units', nargs='*', default='50 20'.split( ), metavar='L',
```

Comment:
Remove commented code - the uncommented `hidden-units` parameter specification that you included below is better anyway!